src/main/java/top/jsoft/homework/logger/Logger.java

```diff
@@ -0,0 +1,41 @@
+package top.jsoft.homework.logger;
+
+import lombok.Getter;
+
+import java.time.LocalDateTime;
+
+/**
+ * Created by psygrammator
+ * group jsoft.top
+ */
+@Getter
+public abstract class Logger {
+    private final LoggerConfiguration loggerConfiguration;
+
+    public Logger()
+    {
+        LoggerConfigurationLoader loggerConfigurationLoader = LoggerConfigurationLoader.load();
+        String fileName = loggerConfigurationLoader.getString("FILE", "log");
+        LoggingLevel loggingLevel = loggerConfigurationLoader.getEnum("LEVEL", LoggingLevel.class, LoggingLevel.INFO);
+        int maxFileSize = loggerConfigurationLoader.getInt("MAX-SIZE", 20000);
+        String textFormat = loggerConfigurationLoader.getString("FORMAT", "[%s][%s] MSG: %s");
+
+        loggerConfiguration = new LoggerConfiguration(fileName, loggingLevel, maxFileSize, textFormat);
+    }
+
+    public void debug(String msg) {
+        if (loggerConfiguration.getLoggingLevel() == LoggingLevel.DEBUG)
```

Comment:
Why do you use this condition? There are some cases when you call method debug but it is not using a debug level?